src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/ChipClient.kt

```diff
@@ -35,11 +32,7 @@ object ChipClient {
 
   fun getDeviceController(context: Context): ChipDeviceController {
     if (!this::chipDeviceController.isInitialized) {
-      chipDeviceController = ChipDeviceController(
```

Comment:
was this replacement of ChipDeviceController() instantiation intentional?

Reply:
Yes, in old way, PreferencesKeyValueStoreManager,  NsdManagerServiceResolver and ChipMdnsCallbackImpl are defined in ChipDeviceController, so it is reasonable to init in ChipDeviceController. But after we move them to platform, we should init it in platform.


      ChipDeviceController.load()
      AndroidChipPlatform.getInstance().setConfigurationManager(DefaultConfigurationManager(this))
      AndroidChipPlatform.getInstance().setKeyValueStoreManager(chip.platform.PreferencesKeyValueStoreManager(this))
      AndroidChipPlatform.getInstance().setServiceResolver(NsdManagerServiceResolver(this))
      AndroidChipPlatform.getInstance().bleManager = AndroidBLEManager()

Reply:
Yes, I know it is really big, but I did not find anyway to separate them. I tried to add platform/android and its ul test first, but this will cause redefine error in ChipDeviceController, so I have to remove them in on PR. Do you have any suggestions?